pg_search/src/index/reader/index.rs

```diff
@@ -905,3 +943,52 @@ pub(super) fn enable_scoring(need_scores: bool, searcher: &Searcher) -> EnableSc
         EnableScoring::disabled_from_searcher(searcher)
     }
 }
+
+#[derive(Default)]
```

Comment:
Would you mind leaving a TODO pointed to https://github.com/quickwit-oss/tantivy/pull/2681#issuecomment-3340222261 here or elsewhere? Just as a reminder that this works, but isn't _quite_ optimal.